xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -653,7 +653,7 @@ void CRendererBase::ProcessHDR(CRenderBuffer* rb)
     {
       // Switch to SDR rendering
       CLog::LogF(LOGINFO, "Switching to SDR rendering");
-      // not need set color space because is alredy set when swap chain is re-created
+      // not need set color space because is already set when swap chain is re-created
```

Comment:
ok but let's fix the comment not just the typo. I think it should be inside the if (m_AutoSwitchHDR) and something like
// color space already sdr or set by the swap chain re-creation

Reply:
> // color space already sdr or set by the swap chain re-creation

This.

Reply:
done